css/03-generic.css

```diff
@@ -221,3 +221,12 @@ h4 {
 .entry-content .wp-block-pullquote:first-child {
 	margin-top: 0;
 }
+
+/* Link Colors */
```

Comment:
I merged this already, but @zoracreates can you move consolidate these selectors with the big block that starts at line 42?